Formula/xssp.rb

```diff
@@ -0,0 +1,35 @@
+class Xssp < Formula
```

Comment:
Please add these citations. https://github.com/cmbi/xssp#citing-xssp
See other formulae for the citation format in Brewsci/bio.

Reply:
And test that it's correct with `brew cite xssp`.

Reply:
Thank you for your suggestion. I added the citation.
However, I didn't find the command `brew cite <formula>` in my Homebrew environment.

Reply:
The `cite` command is in `brew tap brewsci/base`